packages/p2p/src/components/my-profile/payment-methods/payment-methods-list/cancel-edit-payment-method-modal.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { observer } from 'mobx-react-lite';
+import { useStores } from 'Stores';
+import { Localize } from 'Components/i18next';
+
+const CancelEditPaymentMethodModal = () => {
+    const { my_profile_store } = useStores();
+
+    return (
+        <Modal
+            has_close_icon={false}
+            is_open={my_profile_store.is_cancel_edit_payment_method_modal_open}
+            small
+            title={
+                <Text color='prominent' size='s' weight='bold'>
+                    <Localize i18n_default_text='Cancel your edits?' />
```

Comment:
Is this from the design?

Reply:
Yeap, its according to the design
<img width="1106" alt="Screenshot 2022-04-15 at 4 59 50 PM" src="https://user-images.githubusercontent.com/103016120/164618200-29bccbb1-1cf2-41c7-8740-967a42260e27.png">